### ISSUE TITLE
Test methods without modifier are marked as unused

### ISSUE THREAD

**Issue by Saljack** [apache/netbeans#6557]
### Description

JUnit 5 allows to use methods without any access modifier. Developers do not need to add `public` modifier and test classes are more readable. These test methods without any access modifier and with `@Test` annotation are marked in NetBeans as unused. This is not correct behaviour because these methods are used by the JUnit 5 runner. 
```java
import org.junit.jupiter.api.Test;

class TestClass {
  @Test
  void unusedTestMethod() { // <-- this is marked as unused
     // test code
  }

}
```

It would be really nice to remove this false positive warning or disable some hints only in test files. 

Note:
There are multiple annotations which mark test methods e.g. `@ParamatrizedTest` so it should check if method has anywhere `@Testable` annotation.

### Use case/motivation

The test classes contain many warnings and it is easy to overlook some meaningful warning/error.

### Are you willing to submit a pull request?

No

**Comment by mbien** [comment]
this got likely fixed in NB 20 via #6277

**Comment by Saljack** [comment]
Yes it is fixed
